projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/genetics/attributes.json

```
@@ -0,0 +1,49 @@
+{
+    "name": "蜜蜂属性",
+    "icon": "productivebees:gene",
+    "category": "productivebees:genetics",
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "通过将分离出来的基因放在蜜蜂小食上并将其喂给蜜蜂，可以将蜜蜂的属性从一只转移到另一只。$(br)请参阅有关$(l:genetics/collecting#crystalline)收集基因$(/l)的页面。"
+        },
+        {
+            "type": "patchouli:text",
+            "title": "生产力",
+            "text": "生产力水平较高的蜜蜂每次授粉会产生更多的蜜脾。$(br2)生产力水平分为:$(br)正常$(br)中$(br)高$(br)非常高"
```

```suggestion
            "text": "产量较高的蜜蜂每次授粉会产生更多的蜜脾。$(br2)生产力水平分为：$(br)正常$(br)中$(br)高$(br)很高"
```
全角冒号，本文件中不再指出，记得对照zh_cn中的译文